Gemfile

```
@@ -1,20 +1,34 @@
 source 'https://rubygems.org'
 
-# GitHub Pages公式推奨設定
+# Jekyll と GitHub Pages
+gem 'jekyll', '~> 4.3'
 gem 'github-pages', group: :jekyll_plugins
```

RuboCop (1.80.2)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `github-pages` should appear before `jekyll`.

(Bundler/OrderedGems)